src/components/header-top.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+//import { Link } from "react-router-dom";
+import { Nav, NavbarCollapse, NavbarToggle, Navbar, NavbarBrand, NavbarHeader, NavItem, NavDropdown, MenuItem, } from 'react-bootstrap';
```

Comment:
Any reason for adding react-bootstrap rather than using pure bootstrap?

Reply:
Bootstrap wasn't working as expected and I learned bootstrap 4 is not supported by React yet. That is why I used react-bootstrap

Reply:
What issue were you facing ? and any link to the reason why they don't work together?

Reply:
The button wasn't opening when I click on it

https://www.quora.com/Why-do-we-need-React-Bootstrap-when-we-can-use-Bootstrap-in-the-React-app-What-are-the-benefits-of-React-Bootstrap-Id-prefer-to-use-the-original-because-I-want-to-use-Bootstrap-4

https://stackoverflow.com/questions/38015494/can-we-use-pure-bootstrap-with-react-js

That was I decided to use react-bootstrap

Reply:
Okay good stuff